spacelift/data_github_enterprise_integration.go

```diff
@@ -63,10 +114,22 @@ func dataGithubEnterpriseIntegrationRead(ctx context.Context, d *schema.Resource
 		return diag.Errorf("github enterprise integration not found")
 	}
 
-	d.SetId("spacelift_github_enterprise_integration_id") // TF expects id to be set otherwise it will fail
```

Comment:
This _could_ be a breaking change but since this is just a data source I don't think this actually matters or breaks anything